test/findOr.js

```diff
@@ -0,0 +1,66 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('findOr', function () {
+  context('given a list that has the value to find', function () {
+    context('and that value is truthy', function () {
+      specify('should return the found value', function () {
+        const defaultValue = 'default';
+        const list = [1, 2, 3];
+        const predicate = (val) => val === 1;
+
+        assert.deepEqual(RA.findOr(defaultValue, predicate, list), 1);
```

Comment:
```suggestion
        assert.strictEqual(RA.findOr(defaultValue, predicate, list), 1);
```

Reply:
Deep equality of used for asserting equality on arrays or objects, not on primitive values